services/src/main/java/io/druid/cli/PullDependencies.java

```diff
@@ -64,8 +64,7 @@
   private static final Logger log = new Logger(PullDependencies.class);
 
   private static final Set<String> exclusions = Sets.newHashSet(
-      "io.druid",
-      "com.metamx.druid",
```

Comment:
I propose changing the recommendation to be that extensions set the scope of these group ids to `provided` instead of forcefully excluding them here.